### DISCUSSION TITLE
Unifi LXC is not updating to latest 8.0.x version

### DISCUSSION THREAD

**Discussion by thiagohflima** [tteck/Proxmox#2081] [Help Needed] [answered]
As described in the title, after running apt update && apt upgrade, there aren't any new updates, and it is stuck at 7.5.187, which is the latest version within the 7.5.x series.

As far as I remember, it was updating flawlessly through patches from 7.5.x.

**Comment by tteck** [comment, marked as answer]
It hasn't landed in their repository yet.

If you can't wait
```
wget -q https://dl.ui.com/unifi/8.0.7/unifi_sysvinit_all.deb
dpkg -i unifi_sysvinit_all.deb
rm unifi_sysvinit_all.deb
```

**Reply by thiagohflima** [reply]
It makes complete sense. Ty in advance!

**Comment by pabloaugusto** [comment]
Is there a deadline to add update script?
